public/assets/base.css

```diff
@@ -44,3 +44,11 @@
   font-weight: 400;
   font-style: normal;
 }
+
+#splash-screen {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+}
```

Comment:
<img width="61" alt="image" src="https://github.com/leather-wallet/extension/assets/1618764/023c8eac-6c58-4fa5-ab34-c273e481116b">
I thought our editorconfig would enforce this new line? You have the vscode plugin?

Reply:
ah, no I haven't installed plugin. fixed